plugin/symbols/symbols.talon

```diff
@@ -12,19 +12,19 @@ empty escaped (dub string | dub quotes): user.insert_between('\\"', '\\"')
 empty string: user.insert_between("'", "'")
 empty escaped string: user.insert_between("\\'", "\\'")
 (inside parens | args): user.insert_between("(", ")")
-inside (squares | square brackets | list): user.insert_between("[", "]")
-inside (bracket | braces): user.insert_between("{", "}")
+inside (squares | brackets | list): user.insert_between("[", "]")
```

Comment:
I'd recommend keeping "square brackets" here.

Reply:
Done in https://github.com/talonhub/community/pull/1344/commits/f0dbeae39e4f65a7b00fb070efa896af4f805cc0.